### DISCUSSION TITLE
Keycloak not comming up with POD Security

### DISCUSSION THREAD

**Discussion by abalfazlmeshki** [keycloak/keycloak#28910] [Q&A]
Hi friends,

I have deployed Keycloak on a namespace with "pod-security.kubernetes.io/enforce: restricted" label.
The Cluster is version 1.25
I have tried to change keys of "containerSecurityContext:" but still PODs cannot come up.

I appreciate it if you help me to find a customization to make Keycloak run on this restricted namespace.

**Comment by codespearhead** [comment]
What do the logs say?

**Comment by abalfazlmeshki** [comment]
If I enable POD Security is fails with this error
`Error: Could not find or load main class io.quarkus.bootstrap.runner.QuarkusEntryPoint Caused by: java.lang.ClassNotFoundException: io.quarkus.bootstrap.runner.QuarkusEntryPoint`

**Comment by abalfazlmeshki** [comment]
@codespearhead 
Using this image: quay.io/keycloak/keycloak:24.0.3
I have added these settings to the deployment
`
securityContext:

          allowPrivilegeEscalation: false

          capabilities:
            drop:
            - ALL
          privileged: false
          seccompProfile:
            type: RuntimeDefault`
then the POD keeps crashing

**Comment by abalfazlmeshki** [comment]
@codespearhead 
I hope you are doing just fine.
Just want to ask if there is any update for that or not.

**Reply by shahgnp** [reply]
Did you solve it?

**Reply by abalfazlmeshkiotc** [reply]
> Did you solve it?

Yes, Actually it was the dependency of the K8S node :)
Because for this matter, the container reach the node.
What I did was to change the OS platform and switched to another possible one.
I am not sure which cloud you are on. just test to switch to different OS type.
